avsomr/avsomrlocal.cpp

```diff
@@ -145,6 +145,23 @@ QString AvsOmrLocal::stateString() const
 //   setState
 //---------------------------------------------------------
 
+Ret AvsOmrLocal::stateToRet(const State& st) const
+      {
+      switch (st) {
+            case State::Undefined:    return Ret::UnknownError;
+            case State::NotInstalled: return Ret::LocalNotInstalled;
+            case State::Instaling:    return Ret::LocalInstaling;
+            case State::Ready:        return Ret::Ok;
+            case State::Building:     return Ret::LocalAlreadyBuilding;
+            }
+      UNREACHABLE;
```

Comment:
The compiler would want about unhandled cases

Reply:
It should also warn about unreachable code, were there a default case.

Reply:
Also there's `Q_UNREACHABLE();`

Reply:
> The compiler would want about unhandled cases.  
 
My compiler only show a warning, not an error. Warning is easy to miss. It seems there is a compiler flag so that the absence of case is treated as an error.   I'll look for him later

Reply:
Yes, 'only' a warning. But rest assured, I'd catch it sooner or later ;-)
Also there  `-Werror` (for gcc and friends)

Reply:
Anyway, if you see a warning, fix it here and now. My constant hunt for those warnings is for a reason: to not miss the tree int he forrest!